tf-psa-crypto/core/psa_crypto_ecp.c

```diff
@@ -610,7 +610,7 @@ psa_status_t psa_generate_key_iop_setup(
     (void) operation;
     (void) attributes;
 
-    return PSA_SUCCESS;
+    return PSA_ERROR_NOT_SUPPORTED;
```

Comment:
Please change the other functions that return `psa_status_t` as well.

For `psa_generate_key_iop_get_num_ops`, arguably, returning 0 is a correct implementation as long as we don't do interruptibility.